### PR TITLE
Dbatiste/hybrid cleanup

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

**Comment by dbatiste** [comment]
Is this deprecated (i.e. replaced by the fetch stuff)?

**Comment by ryantmer** [comment]
There appear to be a few users still, but I'd say it's largely been replaced by `d2l-fetch` yeah. I can update the readme to include a note about that, as we probably don't want any new users of this.

File: bower.json [hunk 1]
```diff
@@ -3,38 +3,33 @@
...
+    "CODEOWNERS",
+    "package.json",
+    "polymer.json",
+    "wct.conf.json"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#1.9 - 2",
-    "iron-ajax": "PolymerElements/iron-ajax#1.4 - 2"
+    "polymer": "1.9 - 2",
+    "iron-ajax": "1.4 - 2"
```
Comment (ryantmer) [inline bower.json]: Oh, why does this need the range for this? Is `iron-ajax#^2.0.0` not actually hybrid, but fully Polymer 2?

Comment (dbatiste) [reply bower.json]: Oh I was just trying to avoid any conflicts on the consumer end.

**Pull request merged**